zerver/migrations/0699_alter_realm_topics_policy.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 5.1.7 on 2025-03-24 09:11
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("zerver", "0698_set_default_value_for_realm_topics_policy"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="realm",
+            name="topics_policy",
+            field=models.PositiveSmallIntegerField(default=2),
```

Comment:
It's possible we want to define this in this file and then use that, for readability. We try to avoid ever putting raw integers in these files.
```
class RealmTopicsPolicyEnum(Enum):
    allow_empty_topic = 2
```